File: parte001/ex010_triangulo_equilatero.cpp
// Dado  el  valor  del  lado  en  un  triángulo  equilátero, haga  un  algoritmo  que  obtenga  el 
// perímetro, el valor de la altura y el área del triángulo.

#include <iostream>

using namespace std;

int main()
{
    float lado;
    cout << "Ingrese el valor del lado: ";
    cin >> lado;

    cin.ignore();

    float perimetro = lado * 3;
    float altura = lado * 0.866;
    float area = (lado * altura) / 2;

    cout << "El perimetro del triangulo es: " << perimetro << endl;
    cout << "La altura del triangulo es: " << altura << endl;
    cout << "El area del triangulo es: " << area << endl;
    return 0;
}

File: parte001/ex010_triangulo_equilatero.py
# Dado  el  valor  del  lado  en  un  triángulo  equilátero, haga  un  algoritmo  que  obtenga  el 
# perímetro, el valor de la altura y el área del triángulo.

# Captura de datos:
lado = float(input("Ingrese el valor del lado: "))

# Proceso:
perimetro = lado * 3
altura = lado * 0.866
area = lado ** 2 * 0.433

# Salida de datos:

print("El perímetro del triángulo es: ", perimetro)
print("La altura del triángulo es: ", altura)
print("El área del triángulo es: ", area)
